code/modules/reagents/chemistry/holder/holder.dm

```diff
@@ -532,9 +532,16 @@
 		remove_reagent(reagent.type, transfer_amount)
 		transfer_log[reagent.type] = list(REAGENT_TRANSFER_AMOUNT = transfer_amount, REAGENT_PURITY = reagent.purity)
 
+	//combat log
 	if(transferred_by && target_atom)
-		target_atom.add_hiddenprint(transferred_by) //log prints so admins can figure out who touched it last.
-		log_combat(transferred_by, target_atom, "transferred reagents ([get_external_reagent_log_string(transfer_log)]) from [my_atom] to")
+		var/atom/log_target
+		if(isorgan(target_atom))
+			var/obj/item/organ/organ_item = target_atom
+			log_target = organ_item.owner
+		else
+			log_target = target_atom
```

Comment:
```suggestion
		var/atom/log_target = target_atom
		if(isorgan(target_atom))
			var/obj/item/organ/organ_item = target_atom
			log_target = organ_item.owner
```

I prefer this pattern for this sort of filtering. Cleaner, more clear about the default IMO

Reply:
Last review I promise

Reply:
Yeah I agree that looks nicer. That was my original way doing it, thing is coming back full circle.

The null question is one I'm not entirely sure. Can you transfer a reagent to an organ that's outside of a body? Probably. Does it use this specific proc? (The only way I thought would be a syringe, and that doesn't use this one.)

Think it needs a null check?

idk I'll test it a bit more tomorrow